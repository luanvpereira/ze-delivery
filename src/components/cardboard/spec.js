import React from 'react';
import { mount } from 'enzyme';

import Cardboard from './';

describe('<Cardboard/>', () => {
	it('should render a <Cardboard/>', () => {
		const component = mount(
			<Cardboard className="customClass">
				<p>Corpou</p>
			</Cardboard>
		);

		expect(component).toMatchSnapshot();
	});
});
