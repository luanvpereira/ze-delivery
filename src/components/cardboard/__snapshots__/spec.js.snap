// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Cardboard/> should render a <Cardboard/> 1`] = `
<Cardboard
  className="customClass"
>
  <div
    className="cardboard customClass"
  >
    <p>
      Corpou
    </p>
  </div>
</Cardboard>
`;
